protobuf/lib/src/protobuf/proto3_json_string.dart

```diff
@@ -0,0 +1,140 @@
+part of protobuf;
+
+void _writeToProto3JsonSink<S extends StringSink>(
+    _FieldSet fs, TypeRegistry typeRegistry, S sink) {
+  // TODO: Either add `writeToProto3JsonSink` for well-known types, or fall
```

Comment:
An in-between would be to use the current `toProto3Json` helper, walk the ouput that and send it to the Sink...

Reply:
Yeah I think this makes sense, I just implemented this. Performance should be the same as before.